### PR TITLE
Jenkins 50066/unit test health report builder

### PR DESCRIPTION
**Pull request by loelala**

### CODE REVIEW

File: src/test/java/io/jenkins/plugins/analysis/core/quality/HealthReportBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+import java.util.EnumMap;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Priority;
+import static io.jenkins.plugins.analysis.core.testutil.Assertions.assertThat;
+
+import hudson.model.HealthReport;
+
```
Comment (DarphBobo) [inline src/test/java/io/jenkins/plugins/analysis/core/quality/HealthReportBuilderTest.java:11]: javadoc @author is missing

**Review by DarphBobo — COMMENTED** [review]
Looks good :+1:

File: src/test/java/io/jenkins/plugins/analysis/core/quality/HealthReportBuilderTest.java (hunk 1 above)
Comment (loelala) [reply src/test/java/io/jenkins/plugins/analysis/core/quality/HealthReportBuilderTest.java:11]: done

**Review by DarphBobo — COMMENTED** [review]
:+1:

**Pull request merged**